test/switch.civet

```diff
@@ -222,3 +238,112 @@ describe "switch", ->
         }
       }
     """
+
+  describe "pattern matching", ->
+    testCase """
+      basic
+      ---
+      switch x
+        1
+          console.log "it's one"
+        2
+          console.log "it's two"
+      ---
+      if(x === 1) {
+          console.log("it's one")}
```

Comment:
Note for future improvement: Not a fan of the brace placement here. Maybe `InsertNewline`?

Reply:
I was leaning towards fewer `InsertNewline`s to keep the output closer to the input but I could go either way.